docs/packages/universal-components/contributing.md

```diff
@@ -64,7 +64,13 @@ and it is then easier to test the interactions with the action logger, and play
   ```
 
 - **I started the web UI for the mobile storybook but no stories are displayed.**<br>
-  Make sure that the packager is running (`yarn storybook:mobile:universal-components`), and refresh the application by pressing <kbd>CMD</kbd>+<kbd>R</kbd> on iOS or by pressing <kbd>R</kbd> twice on `Android`.
+  Make sure that the packager is running (`yarn storybook:mobile:universal-components`), and refresh the application by pressing <kbd>CMD</kbd>+<kbd>R</kbd> on `iOS` or by pressing <kbd>R</kbd> twice on `Android` or shake the device and press Reload.
+
+  In case refreshing of the application didn't help, try to connect your phone via USB cable with enabled debugging and run
```

Comment:
You would need to make precise that these next two paragraphs are for Android devices. Looks great otherwise!! Can you please amend your commit (`git add .; git commit --amend --no-edit`) with your changes and then force-push your branch (`git push origin +docs/contributing-update`)?
Congratulations on your first contribution! 🎉

Reply:
According to documentation, the shaking of the device should work on both Android and iOS physical devices, so I've made it more clear. Also I've added headline above the second paragraph, so it's clear that it's meant only for Android physical devices. Thank you for review :slightly_smiling_face: